backend/src/app.module.ts

```diff
@@ -4,6 +4,8 @@ import { AppController } from './app.controller';
 import { AppService } from './app.service';
 import { configuration } from './config/configuration';
 import { KmapNeo4jModule } from './config/neo4j/KmapNeo4jModule';
+import { GraphInfoController } from './graph-info/graph-info.controller';
```

Comment:
Rename `graph-info`folder to `schema` to match the directory structure of the shared folder.